app/src/main/java/com/capstone/event_finder/network/EventViewModel.java

```diff
@@ -32,7 +34,10 @@ public LiveData<List<Event>> getEvents() {
     }
 
     public void refreshEvents() {
-        eventRepository.getEventsFromApi();
+        eventApi.getAPIEvents(events -> {
+            eventRepository.deleteAllEvents();
+            eventRepository.insert(events);
+        });
```

Comment:
❓ Do you also need to also update the `LiveData` list here so the UI updates?

Reply:
I don't believe so since that function puts the events from the API into the repository and the UI gets its info from the repository only (which is the LiveData and observation for changes)